draft-scenarios/clash/astral_run.tex

```diff
@@ -20,13 +20,13 @@ \subsection*{\MakeUppercase{Player Setup}}
 \textbf{Starting Units:}
 
 \begin{itemize}
-  \item A Pack of chosen \bronze\ Units
-  \item A chosen Neutral \bronze\ Unit from the player's Faction
+  \item A Pack of chosen \bronze Units
+  \item A chosen Neutral \bronze Unit from the player's Faction
```

Comment:
```suggestion
  \item A Pack of chosen \bronze\ Units
  \item A chosen Neutral \bronze\ Unit from the player's Faction
```
This is incorrect, this way spaces get lost.
<img width="1213" height="555" alt="image" src="https://github.com/user-attachments/assets/faad4081-ee83-45a6-9ba2-ff995a90c495" />